### ISSUE TITLE
VisCrosshair not working with update 1.3.0

### ISSUE THREAD

**Issue by FrancescoBrunoDev** [f5/unovis#306]
The VisCrosshair is not working any more with update 1.3.0 because:

```
Unovis | Crosshair: Y accessors have not been configured. Please check if they're present in the configuration object
Unovis | Crosshair: X accessor function has not been configured. Please check if it's present in the configuration object
```

The wrong behaviour is also in your website:
https://unovis.dev/docs/auxiliary/Crosshair/

**Comment by reb-dev** [comment]
Hey @FrancescoBrunoDev, thanks for the bug report! The fix is included version `1.3.1` 😊

**Comment by FrancescoBrunoDev** [comment]
Thank you!
